helpers/aave/index.ts

```diff
@@ -0,0 +1,140 @@
+import { BaseAdapter, FetchOptions, IJSON, SimpleAdapter } from "../../adapters/types";
+import * as sdk from "@defillama/sdk";
+import AaveAbis from './abi';
+import { Interface } from 'ethers';
+import { normalizeAddress } from '@defillama/sdk/build/util';
+
+export interface AaveLendingPoolConfig {
+  version: 2 | 3;
+  lendingPoolProxy: string;
+  dataProvider: string;
+}
+
+interface ReserveDataUpdatedEvent {
+  transactionHash: any;
+  reserve: string;
+  blockNumber: number;
+  liquidityIndex: bigint;
+}
+
+// PercentageMath uses 4 decimals: https://etherscan.io/address/0x02d84abd89ee9db409572f19b6e1596c301f3c81#code#F17#L15
+const ReserveFactorDecimals = BigInt(1e4);
+
+// https://etherscan.io/address/0x02d84abd89ee9db409572f19b6e1596c301f3c81#code#F16#L16
+const LiquidityIndexDecimals = BigInt(1e27);
+
+async function getPoolFees(pool: AaveLendingPoolConfig, options: FetchOptions): Promise<{
+  dailyFees: sdk.Balances,
+  dailyRevenue: sdk.Balances,
+}> {
+  const dailyFees = options.createBalances()
+  const dailyRevenue = options.createBalances()
+
+  // get reserve (token) list which are supported by lthe ending pool
+  const reservesList: Array<string> = await options.api.call({ target: pool.lendingPoolProxy, abi: AaveAbis.getReservesList })
+
+  // get reserves configs, mainly reserveFactor values
+  const reserveConfigs = await options.api.multiCall({
+    abi: AaveAbis.getReserveConfiguration,
+    calls: reservesList.map(
```

Comment:
no need for the map here, you can move the target to the top level and  do `calls: reservesList,`